packages/webapp/src/_app/hooks/queries.ts

```diff
@@ -27,7 +52,7 @@ export const queryTreasuryStats = {
 
 export const queryMembersStats = {
     queryKey: "query_member_stats",
-    queryFn: getMembersStats,
+    queryFn: getMemberStats,
 };
```

Comment:
There are two queries exposed here now: `queryMemberStats` and `queryMembersStats`. They both have the same `queryKey` and `queryFn`. And when I tested in non-dev-fixtures mode, the member stats are not loading on the home page.